src/NodeAutoCompleteViewExtension/ViewModels/NodeAutoCompleteBarViewModel.cs

```diff
@@ -307,6 +307,11 @@ internal void ConsolidateTransientNodes()
 
             NodeAutoCompleteUtilities.PostAutoLayoutNodes(node.WorkspaceViewModel.Model, node.NodeModel, transientNodes.Select(x => x.NodeModel), true, true, PortViewModel.PortType, null);
 
+            if (PortViewModel.PortType == PortType.Input)
```

Comment:
I think we don't want to execute the source node. It could be something of high complexity and by "dynamo rules" is not really modified.